docs/source/install.rst

```diff
@@ -63,4 +63,6 @@ Instructions for deploying Masu can be found here: https://github.com/project-ko
 Masu can alternatively be deployed with Docker Compose:
 Follow the steps above to start the Koku API with Docker Compose
 
-Start masu services: ``masu_branch> make docker-up``
+Start masu services: ``masu_branch> make docker-up`` 
```

Comment:
How about this:

Ingesting Data with Masu
----------------------
Masu is a data ingestion application bundled with Koku.
For docker-compose deployments, Masu starts up when ` docker-compose up ` is called
Masu runs by default on port 5000
To trigger Masu to process data, send a GET request to http://<baseUrl>:5000/api/cost-management/v1/download/
You may want see Masu logs using `docker logs koku_worker -f`

Reply:
Additional changes may be made in later pull requests.

Reply:
I think we should probably remove references to Masu all together in the docs; I'm fine if you want to do that in a separate PR or in this one.  While it technically is a separate app, it should conceptually be thought of as part of Koku.  The naming of "masu" in the code is really a holdover from the merger.  For this specific part about ingesting data I think it would be good to label the service running at port 5000 as being a Development-Only/Admin service.

Reply:
@dccurtis i'd love to fix the rest of the Masu documentation in a separate PR, as those changes are out of scope for this PR.
This PR is only to document the /download trigger.